addons/udes_stock_picking_batch/models/res_users.py

```diff
@@ -1,11 +1,12 @@
-from odoo import fields, models
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
 
 
 class ResUser(models.Model):
     _inherit = "res.users"
 
     u_location_category_ids = fields.Many2many(
-        comodel_name="stock.location.category", index=True, string="Location Category"
+        comodel_name="stock.location.category", index=True, string=_("Location Category")
```

Comment:
Do we need _ on fields definition i thought we agreed that will put translate=True where needed?